src/pages/nft/class/_classId.vue

```diff
@@ -258,6 +262,10 @@ export default {
       this.getLIKEPrice(),
     ]);
     this.isOwnerInfoLoading = false;
+    if (this.action === 'collect') {
+      logTrackerEvent(this, 'NFT', 'NFTCollect(NFTWidget)', this.classId, 1);
+      this.handleCollect();
+    }
```

Comment:
I think should add a loading indicator before all fetching and show the collect dialog

Reply:
https://user-images.githubusercontent.com/75730405/188055461-7adf1b2e-e66b-4d16-8830-43f8cc97b3c2.mov